From aa7a3071afe2673c7d7faecf4bdd62d3f3162e84 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 23 Feb 2024 17:09:52 +0100
Subject: [PATCH 0940/1141] drm/stm: lvds: change compatible to
 st,stm32mp25-lvds

Lvds is only support by SOC MP25, compatible should link to
the version of SOC.

Change-Id: Ia8a13a157325bf348a97c0a98ef110828c112911
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/362685
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/gpu/drm/stm/lvds.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/stm/lvds.c
+++ b/drivers/gpu/drm/stm/lvds.c
@@ -1303,7 +1303,7 @@ static int __maybe_unused lvds_runtime_r
 
 static const struct of_device_id lvds_dt_ids[] = {
 	{
-		.compatible = "st,stm32-lvds",
+		.compatible = "st,stm32mp25-lvds",
 		.data = NULL
 	},
 	{ /* sentinel */ }
